src/Components/Modal/Modal.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { observer } from "mobx-react";
+import { Button, Container, Col, Row } from "react-bootstrap";
+import styles from "./Modal.module.css";
+import type { MediaPresenter } from "../MediaPresenter";
+
+type Props = {
+  mediaPresenter: MediaPresenter;
+};
+
+export const Modal: React.FC<Props> = observer(({ mediaPresenter }) => {
+  const openModal = (): void => {
+    var modal: HTMLElement | null = document.getElementById("magicModal");
+    if (modal != null) {
+      modal.style.display = "block";
+    }
```

Comment:
Rather than changing the CSS like this, you can add the `classnames` package, which enables you to use a class + state, eg.

```
    import * as cn from 'classnames';

   const [modalOpen, setModalOpen] = React.useState(false);

   const openModal = () => setModalOpen(true);
   const closeModal = () => setModalOpen(false);
  
   ....

   return (
      <div className={cn(styles.modal, { [styles.open]: modalOpen }) }>
        {...}
      </div>
   )
```

--

This here is a place where you could use react state rather than mobx state, because the state change is only presentational, and is contained to within this specific component. If for some reason, you needed to be aware of the state of the modal being open or not outside of `Modal.tsx`, then the state should be controlled externally. However, at current, the open/close is contained within the component, and only adds/removes a CSS class - therefore it's a good candidate for React state